tests/fuzzer.c

```diff
@@ -12,6 +12,7 @@
 /*-************************************
 *  Compiler specific
 **************************************/
+#include "zstd_internal.h"
```

Comment:
minor : document a reason for this additional dependency.

Question : why does it have to be here, rather than in the `*  Includes` paragraph below ?

Reply:
Ohhh, that is why I ended up with duplicate includes... I think vscode decided to start silently adding missing includes...